Source/ConversationV1/Models/ConversationModel.swift

```diff
@@ -0,0 +1,36 @@
+/**
+ * Copyright IBM Corporation 2016
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ **/
+
+import Foundation
+import Freddy
+
+extension Conversation {
```

Comment:
This is another change we made when merging `rest-kit` to `master`. Since we are creating separate targets for each service, there's no longer a need to place models in an extension. Would you mind moving this model outside of the extension?

The reason we used extensions previously was for namespacing. By including them in extensions, `Dialog.Profile` was distinct from `PersonalityInsights.Profile`, for example. Now that we're using a separate target for each service, though, these conflicts no longer occur. So it's safe (and better Swifty style) to move the models outside of the extensions.